django_select2/__init__.py

```diff
@@ -9,4 +9,4 @@
 
 """
 
-__version__ = "5.11.1"
+__version__ = "5.11.0"
```

Comment:
I had already changed the version back to 5.11.0 after you asked me to drop the change some comments ago. Maybe I am not understanding your request, sorry!